openmetadata-spec/src/main/resources/json/schema/metadataIngestion/databaseServiceProfilerPipeline.json

```diff
@@ -35,10 +37,15 @@
       "default": true
     },
     "profileSample": {
-      "description": "Percentage of data used to execute the profiler for the whole workflow. This percentage will be applied to all tables in the workflow. Represented in the range (0, 100].",
+      "description": "Percentage of data we want to execute the profiler and tests on. Represented in the range (0, 100).",
       "type": "number",
-      "exclusiveMinimum": 0,
```

Comment:
being a % should we keep the exclusion? I believe old types were OK

Reply:
ohh my bad, i forgot to revert the exclusive and max thingy